courses/free-courses-hi.md

```diff
@@ -727,6 +727,7 @@
 
 ### Ruby
 
+* [Complete Ruby Programming Course \|\| Hindi](https://www.youtube.com/results?search_query=Ruby+Course+in+Hindi) - CS Geeks
```

Comment:
For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes